deliver/lib/deliver/runner.rb

```diff
@@ -135,7 +137,13 @@ def verify_version
 
     # Upload all metadata, screenshots, pricing information, etc. to App Store Connect
     def upload_metadata
-      upload_metadata = UploadMetadata.new(options)
+      # App clip experience metadata upload must happen before the upload metadata step. The app
+      # clip app store review detail upload depends on there being a valid app clip default
+      # experience on the edit version.
+      UploadAppClipDefaultExperienceMetadata.new.upload_metadata(options)
+      UploadAppClipDefaultExperienceHeaderImages.new.find_and_upload(options)
+
+      upload_metadata = UploadMetadata.new
```

Comment:
Missing arguments `UploadMetadata.new(options)` here, otherwise it fails to initialise.